packages/client/hmi-client/src/components/workflow/ops/compare-datasets/tera-compare-datasets-drilldown.vue

```diff
@@ -182,17 +205,24 @@ const isATESelected = ref(false);
 
 const compareCharts = ref<any[]>([]);
 
+const { activeChartSettings, chartSettings, selectedVariableSettings, removeChartSettings, updateChartSettings } =
+	useChartSettings(props, emit);
+
+const selectedCharts = computed(() => {
+	const selectedChartIds = selectedVariableSettings.value.map((setting) => setting.selectedVariables[0]);
+	return compareCharts.value.filter((chart) => selectedChartIds.includes(chart.title.text));
```

Comment:
Is there specific reason we are maintaining another list of charts, `compareCharts`? I think we should be able to just use existing `useVariableCharts` in `useCharts.ts` with `selectedVariableSettings`